### PR TITLE
Move date stamp banner for URN flow to confirm details page

### PR DESCRIPTION
**Pull request by arthurashman**

### CODE REVIEW

**Comment by arthurashman** [comment]
I think I need to talk through the logic of the datestamp to make sure the right users are seeing it - there's a lot going on in the POST ioj route that I think this PR may circumnavigate

File: app/routes.js [hunk 1]
```diff
@@ -290,15 +298,25 @@ router.get('/case_details', function (req, res) {
...
+  let passported = passporting.isPassported(req.session.data);
 
-  res.render('case_details_offence', { offences: offencesList });
+  res.render('case_details_offence', { offences: offencesList, date_stamp: passported });
 });
 
 router.get('/application_cert_review', function (req, res) {
-  let passported = passporting.isPassported(req.session.data);
   let date = utils.constructDate(req.session.data.dob);
-  res.render('application_cert_review', { date_of_birth: utils.formatDate(date), date_stamp: passported });
```
Comment (oliverweighell) [inline app/routes.js:301]: Looks like we're moving the date stamp forward (to case_details_offence) in the MVP journey? Will this still work where the user goes down case_details_urn path? (ie will the user see the date stamp banner on that journey?)

File: app/views/case_details_offence.html [hunk 2]
```diff
@@ -13,22 +13,22 @@
...
+          Success
+        </h2>
+      </div>
+      <div class="govuk-notification-banner__content">
+        <h3 class="govuk-notification-banner__heading">
+          Your application has been date stamped
+        </h3>
+      </div>
+    </div>
+  {% endif %}
```
Comment (oliverweighell) [inline app/views/case_details_offence.html:29]: We could move the date stamp snippet into an 'include' that can then be called into the various pages it's needed? Eg see /views/includes/head.html which is called into layout.html.

File: app/routes.js (hunk 1 above)
Comment (oliverweighell) [reply app/routes.js:301]: Ah yes, date stamp would be shown on case_details_confirm for urn path?

**Review by oliverweighell — CHANGES_REQUESTED** [review]

**Comment by oliverweighell** [comment]
> 

Agree, be good to walkthrough, the logic has changed a fair bit (the logic you quote is where IOJ section is complete and the app is non-passported)

**Pull request merged**